examples/android-proguard-example/README.md

```diff
@@ -0,0 +1,9 @@
+# android-proguard-example
+
+Example Android project showing how to properly configure [ProGuard](https://www.guardsquare.com/proguard).
+ProGuard is a tool for 'shrinking' and obfuscating compiled classes, it removes methods and fields if
```

Comment:
This sentence could perhaps be improved. What about this?

> ProGuard is a tool for 'shrinking' and obfuscating compiled classes.
> It can rename methods and fields, or remove them if they appear to be unused.

Reply:
That sounds better, thanks!